js/questions.js

```diff
@@ -0,0 +1,243 @@
+var totalScore = {
+  extraversion: 0,
+  introversion: 0,
+  sensing: 0,
+  intuition: 0,
+  thinking: 0,
+  feeling: 0,
+  judging: 0,
+  perceiving: 0,
+}
+
+var fullQuestions = {
```

Comment:
It makes more sense, since you are referencing the questions with a number anyways, to keep them in an array:
```js
var questions = [
  {
    question: "You are almost never late for your appointments",
    option1: { type: "feeling",  point: 2 },
    option2: { type: "feeling",  point: 1 },
    option3: { type: "none",     point: 0 },
    option4: { type: "thinking", point: 1 },
    option5: { type: "thinking", point: 2 }
  },
  {
    question: "You like to be engaged in an active and fast-paced job.",
    option1: { type: "feeling",  point: 2 },
    option2: { type: "feeling",  point: 1 },
    option3: { type: "none",     point: 0 },
    option4: { type: "thinking", point: 1 },
    option5: { type: "thinking", point: 2 }
  },
```